templates/repo/view_file.tmpl

```diff
@@ -46,7 +46,10 @@
 			{{end}}
 			{{if not .ReadmeInList}}
 				<div class="ui buttons mr-2">
-					<a class="ui mini basic button" href="{{$.RawFileLink}}">{{.locale.Tr "repo.file_raw"}}</a>
+					<button class="ui mini basic button small compact tooltip" id="clipboard-btn" data-content="{{.locale.Tr "copy"}}" data-fetch-text="true" data-clipboard-target="#raw-file-link" aria-label="{{.locale.Tr "copy"}}">
```

Comment:
That `Copy` translation is insufficient:
Do you copy the *file name* or the *file content*?
As the copy button is positioned in the header, it could be both.
Better not let users guess which one it'll be.

Reply:
Also, doesn't tippy already take care of accessibility on elements that have tooltips?
Or did I imagine that?

Reply:
Would label it "Copy content to clipboard".

No, I don't think we do any accessibilty to `data-content` tooltips. Using `aria-label` would be nice but it can not unconditionally be added to all buttons. Could added via JS if the have a `<svg>` as their only child.

Reply:
Do we even need the `to clipboard`? Where else could you copy it?
I think that causes more confusion since many users likely don't know what that is.

Reply:
Indeed, `Copy Content` is sufficient.

Reply:
Switched to "Copy content"